surface/frontend/src/gamepad/bettergamepad.js

```diff
@@ -18,6 +18,27 @@ var gp = {
     but_func: {},
     ax_func: {},
 
+
+    freeze() {
+      var cur = navigator.getGamepads()[gp.gamepadIndex];
+      var lay = layouts[gp.layoutKey];
+      var val = cur[buttn.where][buttn.indx];
```

Comment:
`bttn.where`?
I would also like to know where the button comes from here
Cause right now it's undefined